draft-ietf-tls-tls13.md

```diff
@@ -3488,6 +3503,9 @@ access_denied
 decode_error
 : A message could not be decoded because some field was out of the
   specified range or the length of the message was incorrect.
+  This alert is used for errors where the message does not conform
+  to the formal protocol syntax (e.g., a vector is specified as
+  having the range <2..255> but is encoded as having 0 length.)
```

Comment:
not insisting on it, but maybe also add "or a struct specified as having 4 fields is received with only 3 fields present"?